velox/docs/functions/presto/conversion.rst

```diff
@@ -314,12 +314,19 @@ Invalid examples if cast_to_int_by_truncate=false
 
   SELECT cast('12345.67' as tinyint); -- Invalid argument
   SELECT cast('1.2' as tinyint); -- Invalid argument
+  SELECT cast('1.0' as tinyint); -- Invalid argument
```

Comment:
Please, include English description of this behavior as well.

Reply:
Done.